docs/about/contributing.md

```diff
@@ -50,8 +50,9 @@ to the git repository.
 ## Running the tests
 
 To run the tests, it is recommended that you use [Tox]. This just needs
-to be pip installed and then the test suite can be ran for MkDocs but running
-the command `tox` in the root of your MkDocs repository.
+[pip](https://pypi.org/project/pip/) to be installed and then the test suite
```

Comment:
"needs to be pip installed" is not an error. In other words, "`tox` needs to be installed using pip." Or "run the command `pip install tox`."

I suppose this could be worded more clearly, but this change is not correct as it alters the intended meaning.

Reply:
My mistake (I misread that). Fixed in 8fe2c33.